### ISSUE TITLE
A11y_VscodeWebviewUiToolkit_AllComponents_DataGrid_DevTools: No name property is provided for any of the data grids

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/vscode-webview-ui-toolkit#183]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_VscodeWebviewUiToolkit_Aug2021;#Webview UI Toolkit for Visual Studio Code;#Win32;#FTP;#DesktopApp;#WCAG4.1.2;#AINotInScope;

### Environment Details:
Application Name: Vscode Webview Ui Toolkit
Windows Version: Win10

### Repro Steps:

1. Launch VS Code
2. Open WebView UI Toolkit: All Component extension
3. Tab to any Data Grid open developer tool by hitting "ctrl+shift+i" and check if name property is provided for any data grids

### Actual:
No name property is provided for any of the data grids like "default data grid" etc.

Similar issue is with all the "Panels" in the "Panels" scenario.

### Expected:
The unique name property should be provided for all the data grids.

### User Impact:
Screen reader users will not get the proper information about the data grids

**Comment by fsteffi** [comment]
**GithubTags:**#Closed;
